tests/unit_tests/pyanaconda_tests/modules/payloads/payload/test_module_payload_dnf_manager.py

```diff
@@ -927,6 +927,24 @@ def test_repositories(self):
 
         assert self.dnf_manager.repositories == ["r1", "r2", "r3"]
 
+    def test_enabled_repositories(self):
+        """Test the enabled_repositories property."""
+        assert self.dnf_manager.enabled_repositories == []
+
+        r1 = self._add_repo("r1")
+        r2 = self._add_repo("r2")
+        r3 = self._add_repo("r3")
+        r4 = self._add_repo("r4")
+
+        assert self.dnf_manager.repositories == ["r1", "r2", "r3", "r4"]
```

Comment:
Do you perhaps also want to test the initial state ({dis|en}abled) after adding?

Reply:
Do you mean the initial state of the added repositories? Doesn't this check test their state?

Reply:
Ah, sorry, I left out too much, again :( So. You do this:
1. Add repos
2. Test repo presence
3. Set enabled/disabled
4. Test enabled/disabled

IMO from the test's point of view, the enabled/disabled state after adding is undefined. So I'm asking if this makes sense instead:
1. Add repos
2. Test repo presence
3. *Test enabled/disabled*
4. Set enabled/disabled
5. Test enabled/disabled

Reply:
I see. Actually, I didn't like that test, because it tested DNF instead of Anaconda, so I have removed it.